config/config.nonprod.yaml

```diff
@@ -1,14 +1,14 @@
 ---
 # Action Config
 - whiteboard_tag: nonprodtest
-  contact: bsieber@mozilla.com
+  contact: 644672
```

Comment:
Couldn't we have sub-fields instead? 
Like
```yaml
  contact:
    email: bsieber@mozilla.com
    bugzilla: 644672
```

Reply:
We could, but emails are PII and in the linked bug, Frida suggested to modify it to bugzilla user IDs

Reply:
Ok then.
Yes, renaming the field would definitely help.
`bugzilla_user_id` is a bit ugly compared to `contact` but at least it would be more explicit for readers/contributors. There could be nicer alternative names! 🤷

Reply:
Updated from `contact` to `bugzilla_user_id` and updated the readme